lte/gateway/c/oai/lib/s6a_proxy/itti_msg_to_proto_msg.cpp

```diff
@@ -90,6 +90,13 @@ UpdateLocationRequest convert_itti_s6a_update_location_request_to_proto_msg(
     ret.set_skip_subscriber_data(SKIP_SUBSCRIBER_DATA);
   }
 
+  /*
+   * Set the dual_registeration_5g_indicator
+   */
+  if (msg->dual_regis_5g_ind) {
```

Comment:
@Vyas20 , this PR does not include how the `dual_regis_5g_ind` value is set in the `msg`. It needs to be set by MME here https://github.com/magma/magma/blob/master/lte/gateway/c/oai/tasks/nas/emm/Authentication.c#L1452 based on the flags seen in the Attach Request message

Reply:
@Vyas20 can you address/respond to @ssanadhya's comment?

Reply:
Added flag nr_as_secondary_rat in attach request message as per  3GPP 29.272 V-15.10.0.